nishimweelysee/Elysee-CapstoneProject
# Ft list articles

#### What does this PR do?
> This PR Used to display all article to the admin so that he can navigate into and see their information without memorizing their ids.
>
#### Description of Task to be completed
>- There is an image of article
>- There is a title of article
>
#### How should this be manually tested?
>When you open a website and go to admin pages and go into blogs or portfolio you will see list of all articles.
#### Any background context you want to provide?
>
#### What are the relevant Trello story?
>FT(View List of all articles) Use this link to visit my Trello board 👍 https://trello.com/c/kxwSrPZM
#### Screenshots (if appropriate)
>
> This is for Blog posts
>
>![image](https://user-images.githubusercontent.com/56052588/92328851-e30e1180-f063-11ea-95ce-81069ccb8039.png)
>
>This is for Portfolio Posts
>
>
![image](https://user-images.githubusercontent.com/56052588/92328867-1650a080-f064-11ea-87ce-654fe1cf407e.png)
>
> Thank you !!
